aws/aws-how-to/instances/create-ec2-using-ubuntu-image.rst

```diff
@@ -0,0 +1,112 @@
+How to create an EC2 instance using an Ubuntu image
+===================================================
+
+This page gives you the first keys to using official Ubuntu Cloud images on Amazon EC2. Please follow the instructions below to create EC2 instance with Ubuntu Cloud images.
+
+Running Ubuntu on Amazon Web Services requires you to go through the following steps: 
+
+1. Create your account on AWS (if you do not already have one) and setup your credentials
+2. Instantiate your images(s)
+3. Configure your instance 
```

Comment:
Change these to:
3. Find an Ubuntu AMI
4. Create a security group (Optional)
5. Instantiate the image